sonar/config.py

```
@@ -121,8 +121,10 @@ def _(x):
 #: proxies) removes these headers again before sending the response to the
 #: client. Set to False, in case of doubt.
 ACCOUNTS_USERINFO_HEADERS = True
-# make security blueprints available to the REST API
-ACCOUNTS_REGISTER_BLUEPRINT = True
+#: Path of the reset password page, on the UI application. Used by the API
+#: application to build the links sent by email, as the Flask-Security views are
+#: only registered on the UI application.
+ACCOUNTS_REST_RESET_PASSWORD_ENDPOINT = "/lost-password/{token}/"
```

Ruff (0.16.0)

[error] Possible hardcoded password assigned to: "ACCOUNTS_REST_RESET_PASSWORD_ENDPOINT"

(S105)